<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoshopLike</class>
 <widget class="QMainWindow" name="PhotoshopLike">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="DrawCanvas" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="createImage"/>
    <addaction name="openImage"/>
    <addaction name="separator"/>
    <addaction name="saveAs"/>
    <addaction name="save"/>
    <addaction name="separator"/>
    <addaction name="action7"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="undo"/>
    <addaction name="redo"/>
   </widget>
   <widget class="QMenu" name="menu_I">
    <property name="title">
     <string>图像(&amp;I)</string>
    </property>
    <widget class="QMenu" name="menu_G">
     <property name="title">
      <string>画像旋转(G)</string>
     </property>
     <addaction name="rotate180"/>
     <addaction name="rotate90"/>
     <addaction name="rotate270"/>
     <addaction name="separator"/>
     <addaction name="flipHorizontal"/>
     <addaction name="flipVertical"/>
    </widget>
    <widget class="QMenu" name="menu_J_2">
     <property name="title">
      <string>调整(J)</string>
     </property>
     <addaction name="HSL"/>
     <addaction name="cb"/>
     <addaction name="separator"/>
     <addaction name="gray"/>
     <addaction name="inverse"/>
    </widget>
    <addaction name="menu_J_2"/>
    <addaction name="separator"/>
    <addaction name="imageSize"/>
    <addaction name="canvasSize"/>
    <addaction name="menu_G"/>
    <addaction name="action3_4"/>
   </widget>
   <widget class="QMenu" name="menu_L">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
   <addaction name="menu_I"/>
   <addaction name="menu_L"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="Tools" name="Tool"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="LayerGroup" name="layerGroup"/>
  </widget>
  <action name="createImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>新建(N)</string>
   </property>
  </action>
  <action name="openImage">
   <property name="text">
    <string>打开(O)</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>关闭(C)</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>存储(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action6">
   <property name="text">
    <string>导入(M)</string>
   </property>
  </action>
  <action name="action7">
   <property name="text">
    <string>退出(X)</string>
   </property>
  </action>
  <action name="undo">
   <property name="text">
    <string>还原(Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redo">
   <property name="text">
    <string>重做(Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action3_2">
   <property name="text">
    <string>剪切(T)</string>
   </property>
  </action>
  <action name="action4_2">
   <property name="text">
    <string>拷贝(C)</string>
   </property>
  </action>
  <action name="action5_2">
   <property name="text">
    <string>合并拷贝(Y)</string>
   </property>
  </action>
  <action name="action6_2">
   <property name="text">
    <string>粘贴(P)</string>
   </property>
  </action>
  <action name="action7_2">
   <property name="text">
    <string>清除(E)</string>
   </property>
  </action>
  <action name="action8">
   <property name="text">
    <string>键盘快捷键和菜单</string>
   </property>
  </action>
  <action name="action9">
   <property name="text">
    <string>菜单(U)</string>
   </property>
  </action>
  <action name="action10">
   <property name="text">
    <string>自定义工具栏</string>
   </property>
  </action>
  <action name="action11">
   <property name="text">
    <string>首选项(N)</string>
   </property>
  </action>
  <action name="action3_3">
   <property name="text">
    <string>自动色调(N)</string>
   </property>
  </action>
  <action name="action4_3">
   <property name="text">
    <string>自动对比度(U)</string>
   </property>
  </action>
  <action name="action5_3">
   <property name="text">
    <string>自动颜色(O)</string>
   </property>
  </action>
  <action name="imageSize">
   <property name="text">
    <string>图像大小(I)</string>
   </property>
  </action>
  <action name="canvasSize">
   <property name="text">
    <string>画布大小(S)</string>
   </property>
  </action>
  <action name="action3_4">
   <property name="text">
    <string>裁剪(P)</string>
   </property>
  </action>
  <action name="action4_4">
   <property name="text">
    <string>裁切(R)</string>
   </property>
  </action>
  <action name="action1_5">
   <property name="text">
    <string>显示全部(V)</string>
   </property>
  </action>
  <action name="action1_6">
   <property name="text">
    <string>位图(B)</string>
   </property>
  </action>
  <action name="action2_5">
   <property name="text">
    <string>灰度(G)</string>
   </property>
  </action>
  <action name="action3_5">
   <property name="text">
    <string>双色调(D)</string>
   </property>
  </action>
  <action name="action4_5">
   <property name="text">
    <string>索引颜色(I)</string>
   </property>
  </action>
  <action name="action5_4">
   <property name="text">
    <string>RGB颜色(R)</string>
   </property>
  </action>
  <action name="action6_4">
   <property name="text">
    <string>CMYK颜色(C)</string>
   </property>
  </action>
  <action name="action7_3">
   <property name="text">
    <string>Lab颜色(L)</string>
   </property>
  </action>
  <action name="action8_2">
   <property name="text">
    <string>多通道(M)</string>
   </property>
  </action>
  <action name="action9_2">
   <property name="text">
    <string>8位/通道(A)</string>
   </property>
  </action>
  <action name="action10_2">
   <property name="text">
    <string>16位/通道(N)</string>
   </property>
  </action>
  <action name="action11_2">
   <property name="text">
    <string>32位/通道(H)</string>
   </property>
  </action>
  <action name="action12">
   <property name="text">
    <string>颜色表(T)</string>
   </property>
  </action>
  <action name="action1_7">
   <property name="text">
    <string>亮度/对比度(C)</string>
   </property>
  </action>
  <action name="action2_6">
   <property name="text">
    <string>色阶(L)</string>
   </property>
  </action>
  <action name="action3_6">
   <property name="text">
    <string>曲线(U)</string>
   </property>
  </action>
  <action name="action4_6">
   <property name="text">
    <string>曝光度(E)</string>
   </property>
  </action>
  <action name="action5_5">
   <property name="text">
    <string>自然饱和度(V)</string>
   </property>
  </action>
  <action name="action6_5">
   <property name="text">
    <string>色相/饱和度(H)</string>
   </property>
  </action>
  <action name="action78">
   <property name="text">
    <string>色彩平衡(B)</string>
   </property>
  </action>
  <action name="action8_3">
   <property name="text">
    <string>黑白(K)</string>
   </property>
  </action>
  <action name="action9_3">
   <property name="text">
    <string>照片滤镜(F)</string>
   </property>
  </action>
  <action name="action10_3">
   <property name="text">
    <string>通道混合器(X)</string>
   </property>
  </action>
  <action name="action11_3">
   <property name="text">
    <string>颜色查找</string>
   </property>
  </action>
  <action name="action12_2">
   <property name="text">
    <string>反相(I)</string>
   </property>
  </action>
  <action name="action13">
   <property name="text">
    <string>色调分离(P)</string>
   </property>
  </action>
  <action name="action14">
   <property name="text">
    <string>阈值(T)</string>
   </property>
  </action>
  <action name="action15">
   <property name="text">
    <string>渐变映射(G)</string>
   </property>
  </action>
  <action name="action16">
   <property name="text">
    <string>可选颜色(S)</string>
   </property>
  </action>
  <action name="action17">
   <property name="text">
    <string>阴影/高光(W)</string>
   </property>
  </action>
  <action name="action18">
   <property name="text">
    <string>HDR色调</string>
   </property>
  </action>
  <action name="action19">
   <property name="text">
    <string>去色(D)</string>
   </property>
  </action>
  <action name="action20">
   <property name="text">
    <string>匹配颜色(M)</string>
   </property>
  </action>
  <action name="action21">
   <property name="text">
    <string>替换颜色(R)</string>
   </property>
  </action>
  <action name="action22">
   <property name="text">
    <string>色调均化(Q)</string>
   </property>
  </action>
  <action name="rotate180">
   <property name="text">
    <string>180度(1)</string>
   </property>
  </action>
  <action name="rotate90">
   <property name="text">
    <string>顺时针90度(9)</string>
   </property>
  </action>
  <action name="rotate270">
   <property name="text">
    <string>逆时针90度(0)</string>
   </property>
  </action>
  <action name="rotateAny">
   <property name="text">
    <string>任意角度(A)</string>
   </property>
  </action>
  <action name="flipHorizontal">
   <property name="text">
    <string>水平翻转画布(H)</string>
   </property>
  </action>
  <action name="flipVertical">
   <property name="text">
    <string>垂直翻转画布(V)</string>
   </property>
  </action>
  <action name="action1_9">
   <property name="text">
    <string>快速导出为PNG</string>
   </property>
  </action>
  <action name="action2_8">
   <property name="text">
    <string>导出为</string>
   </property>
  </action>
  <action name="action3_8">
   <property name="text">
    <string>导出首选项</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="text">
    <string>另存为(&amp;A)……</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="HSL">
   <property name="text">
    <string>色相/饱和度/明度</string>
   </property>
  </action>
  <action name="gray">
   <property name="text">
    <string>转为灰度图</string>
   </property>
  </action>
  <action name="inverse">
   <property name="text">
    <string>反色</string>
   </property>
  </action>
  <action name="cb">
   <property name="text">
    <string>亮度/对比度</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Tools</class>
   <extends>QWidget</extends>
   <header>tools.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DrawCanvas</class>
   <extends>QWidget</extends>
   <header>drawcanvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayerGroup</class>
   <extends>QWidget</extends>
   <header>layergroup.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
